{
    "type": "bundle",
    "id": "bundle--f6b7acca-a27d-41d4-bee2-d23340fc15d5",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--0ed1db6f-a477-44ce-ad05-9a568f85633e",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-02-27T01:12:56.555964Z",
            "modified": "2025-02-27T01:12:56.555964Z",
            "name": "CVE-2022-49388",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nubi: ubi_create_volume: Fix use-after-free when volume creation failed\n\nThere is an use-after-free problem for 'eba_tbl' in ubi_create_volume()'s\nerror handling path:\n\n  ubi_eba_replace_table(vol, eba_tbl)\n    vol->eba_tbl = tbl\nout_mapping:\n  ubi_eba_destroy_table(eba_tbl)   // Free 'eba_tbl'\nout_unlock:\n  put_device(&vol->dev)\n    vol_release\n      kfree(tbl->entries)\t  // UAF\n\nFix it by removing redundant 'eba_tbl' releasing.\nFetch a reproducer in [Link].",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-49388"
                }
            ]
        }
    ]
}